src/frontend/components/DashboardVideoLiveControlPane/tab/DashboardVideoLiveTabAttendance/index.tsx

```diff
@@ -0,0 +1,255 @@
+  },
+});
+
+interface DashboardVideoLiveTabAttendanceProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveTabAttendance = ({
+  video,
+}: DashboardVideoLiveTabAttendanceProps) => {
+  const intl = useIntl();
+  const interval =
+    video.live_state &&
+    [liveState.STOPPING, liveState.RUNNING].includes(video.live_state)
+      ? POLL_FOR_ATTENDANCES
+      : false;
+  // only if the video is running or is stopping, we refresh the list
+  const { data, status } = useLiveAttendances({ refetchInterval: interval });
+
+  switch (status) {
+    case 'idle':
+    case 'loading':
+      return (
+        <Box width="full">
+          <Spinner size="large">
+            <FormattedMessage {...messages.loading} />
+          </Spinner>
+        </Box>
+      );
+    case 'error':
+      return (
+        <Box width="full">
+          <ErrorMessage code="generic" />
+        </Box>
+      );
+
+    case 'success':
+      return (
+        <Box width="full">
+          {data?.count > 0 &&
+            data?.results.map((liveSession) => (
+              <DashboardVideoSessionLiveTabAttendance
+                key={liveSession.id}
+                liveSession={liveSession}
+              />
+            ))}
+          {data?.count === 0 && (
+            <Box
+              background={{
+                image: `url(${appData.static.img.liveBackground})`,
+                position: 'top',
+                repeat: 'no-repeat',
+                size: 'cover',
+              }}
+              flex="grow"
+            >
+              <Box
+                margin="auto"
+                pad={{ horizontal: 'none', vertical: 'large' }}
+              >
+                <Box
+                  background="white"
+                  fill
+                  round="6px"
+                  margin={{ bottom: 'small', horizontal: 'auto', top: 'none' }}
+                  pad="large"
+                  style={{
+                    boxShadow: '0px 0px 6px 0px rgba(2, 117, 180, 0.3)',
+                  }}
+                >
+                  <Box margin="auto" pad={{ horizontal: '36px' }}>
+                    <Heading color="blue-active" level={3} margin="auto">
+                      {intl.formatMessage(messages.noParticipants)}
+                    </Heading>
+                  </Box>
+                </Box>
+              </Box>
+            </Box>
+          )}
+        </Box>
+      );
+  }
+};
+
+interface DashboardVideoSessionLiveTabAttendanceProps {
+  liveSession: LiveAttendance;
+}
+export const DashboardVideoSessionLiveTabAttendance = ({
+  liveSession,
+}: DashboardVideoSessionLiveTabAttendanceProps) => {
+  // over 80% of attendance, the user is considered very diligent
+  const diligentLimit = 80;
+  const isConnected = (liveAttendanceInfo: LiveAttendanceInfo) => {
+    return liveAttendanceInfo.onStage ||
+      liveAttendanceInfo.connectedInBetween ||
+      liveAttendanceInfo.playing
+      ? true
+      : false;
+  };
```

Comment:
```suggestion
  const isConnected = (liveAttendanceInfo: LiveAttendanceInfo) => (
    liveAttendanceInfo.onStage ||
    liveAttendanceInfo.connectedInBetween ||
    liveAttendanceInfo.playing
  );
```

Reply:
I didn't want it to be possibly undefined, but I must have restructured the code because it doesn't seem to be a problem anymore